presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -53,12 +58,18 @@
 import static io.prestosql.spi.type.TinyintType.TINYINT;
 import static io.prestosql.spi.type.VarbinaryType.VARBINARY;
 import static io.prestosql.spi.type.VarcharType.VARCHAR;
+import static java.util.Collections.emptyList;
 import static java.util.Objects.requireNonNull;
 
 public class ElasticsearchMetadata
         implements ConnectorMetadata
 {
     private static final String ORIGINAL_NAME = "original-name";
+    public static final String SUPPORTS_PREDICATES = "supports-predicates";
+
+    private static final Set<String> UNSUPPORTED_PREDICATE_COLUMNS = ImmutableSet.of(
+            BuiltinColumns.SCORE.getName(),
+            BuiltinColumns.SOURCE.getName());
```

Comment:
Shouldn't this be populated based on `BuiltinColumns#isSupportsPredicate`?

Reply:
remove `UNSUPPORTED_PREDICATE_COLUMNS`

Reply:
Ah yes, leftover from another experimental approach I was trying.